docs/source/index.rst

```diff
@@ -1,11 +1,12 @@
-Welcome to AA Toolbox
-=====================
+Welcome to OCHA AnticiPy
+========================
 
-Welcome to AA Toolbox's documentation.
+Welcome to OCHA AnticiPy's documentation.
 
-The `anticipatory action <https://centre.humdata.org/anticipatory-action/>`_
-(AA) toolbox is a Python package to support development of AA frameworks, by
-simplifying the downloading and processing of commonly used datasets.
+OCHA AnticiPy is a Python library for simple downloading and processing
```

Comment:
Do we streamline a single definition or type across the doc files? See comments on the others ones.

Reply:
I've now made it consistent throughout.